src/Approximations/overapproximate.jl

```diff
@@ -1635,3 +1635,66 @@ function overapproximate(X::LazySet{N}, ::Type{<:Zonotope},
     end
     return Zonotope(c, G)
 end
+
+"""
+    overapproximate(r::Rectification{N, <:Zonotope{N}}, ::Type{<:AbstractZonotope}) where {N}
```

Comment:
remember to add this new function to the docs

Reply:
is it needed for a multiple dispatch?

Reply:
what i meant is to "register" this function in the appropriate place inside `docs/...`

Reply:
since we build docs with an option like `force=true` this implies that if there is function which has a docstring and it *has not* been added to the docs then they will fail.

Reply:
In this case we only have the function `overapproximate` in the docs (and not each individual method), so it will work.